sdks/jreleaser-tool-java-sdk/src/main/java/org/jreleaser/sdk/tool/AbstractTool.java

```diff
@@ -20,6 +20,7 @@
 import org.jreleaser.bundle.RB;
 import org.jreleaser.model.api.JReleaserContext;
 import org.jreleaser.sdk.command.Command;
+import org.jreleaser.sdk.command.Command.Result;
```

Comment:
This import seems to be extra. Please run `./gradlew check` to find unnecessary imports